README.md

```diff
@@ -23,26 +23,28 @@ With coremltools, you can do the following:
 
 After conversion, you can integrate the Core ML models with your app using Xcode.
 
-## Version 5
```

Comment:
Most GitHub README for most Python package don't mention what's new in the most recent major version. I think it might be better if this whole section of the README was basically just: you can install coremltools by running `pip install coremltools`. What are your thoughts here?

Reply:
I could remove all feature-related content and provide a link to [New in coremltools](https://coremltools.readme.io/docs/new-in-coremltools). I saw that version 5.0 had its features listed, so that's why I added 6.0 features. We should remove the 5.0 features as well, since it is covered in [New in coremltools](https://coremltools.readme.io/docs/new-in-coremltools).

Reply:
That sounds great. Thanks.